src/databricks/sql/thrift_api/TCLIService/constants.py

```diff
@@ -13,57 +13,57 @@
 import sys
 from .ttypes import *
 PRIMITIVE_TYPES = set((
-        0,
-        1,
-        2,
-        3,
-        4,
-        5,
-        6,
-        7,
-        8,
-        9,
-        15,
-        16,
-        17,
-        18,
-        19,
-        20,
-        21,
+    0,
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    15,
+    16,
+    17,
+    18,
+    19,
+    20,
+    21,
 ))
 COMPLEX_TYPES = set((
-        10,
-        11,
-        12,
-        13,
-        14,
+    10,
+    11,
+    12,
+    13,
+    14,
 ))
 COLLECTION_TYPES = set((
-        10,
-        11,
+    10,
+    11,
 ))
 TYPE_NAMES = {
-        10: "ARRAY",
-        4: "BIGINT",
-        9: "BINARY",
-        0: "BOOLEAN",
-        19: "CHAR",
-        17: "DATE",
-        15: "DECIMAL",
-        6: "DOUBLE",
-        5: "FLOAT",
-        21: "INTERVAL_DAY_TIME",
-        20: "INTERVAL_YEAR_MONTH",
-        3: "INT",
-        11: "MAP",
-        16: "NULL",
-        2: "SMALLINT",
-        7: "STRING",
-        12: "STRUCT",
-        8: "TIMESTAMP",
-        1: "TINYINT",
-        13: "UNIONTYPE",
-        18: "VARCHAR",
+    10: "ARRAY",
+    4: "BIGINT",
+    9: "BINARY",
+    0: "BOOLEAN",
+    19: "CHAR",
```

Comment:
Changes like this are fine. The significant number of changed lines are a result of different formatted output from the compiler.